apps/researcher/src/app/[locale]/communities/[slug]/page.tsx

```diff
@@ -10,7 +10,8 @@ import {revalidatePath} from 'next/cache';
 import {objectList} from '@colonial-collections/database';
 import ObjectCard from './object';
 import AddObjectListForm from '@/components/add-object-list-form';
-import {SlideOutButton, SlideOut, Notifications} from 'ui';
+import {SlideOutButton, SlideOut, SlideOutClosed, Notifications} from 'ui';
```

Comment:
Out of curiosity: we make use of the `database` package by importing from `@colonial-collections/database` but we make use of the `ui` package by importing from `ui` (without the `@colonial-collections` prefix). Is there a rule that we should follow?

Reply:
This is because this was the first package I made, and we did not have this convention then. But I can update it so they are all the same.

Reply:
That sounds perfect! Doesn't have to be a part of this PR, though.